server/join.go

```diff
@@ -103,6 +126,9 @@ func PrepareJoinCluster(cfg *Config) error {
 
 	existed := false
 	for _, m := range listResp.Members {
+		if len(m.Name) == 0 {
+			return errors.New("exsist a member that the join is not completed")
```

Comment:
How about `there is a member that has not been joined successfully`?
PTAL @CaitinChen